spec/features/user_signs_up_spec.rb

```diff
@@ -16,8 +16,8 @@
     visit new_user_registration_path
 
     fill_in 'Email', with: 'john@example.com'
-    fill_in 'Password', with: 'password'
-    fill_in 'Password confirmation', with: 'password'
+    fill_in 'Password', with: 'password', match: :prefer_exact
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.